ansible/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Global variables for all nodes
+
+# Paths (relative to playbook execution directory or absolute)
+network_dir: "{{ playbook_dir }}/../local-devnet"
+genesis_dir: "{{ network_dir }}/genesis"
+data_dir: "{{ network_dir }}/data"
+validator_config_file: "{{ genesis_dir }}/validator-config.yaml"
+
+# Genesis generation settings
+pk_docker_image: "ethpandaops/eth-beacon-genesis:pk910-leanchain"
```

Comment:
this is not remotely geneated, but config is always locally generated and copied over.

infact everything is to be locally generated. only in the followup PR we will distribute generation of the validators